storagenode/pieces/store.go

```diff
@@ -62,6 +62,8 @@ type DB interface {
 	Delete(ctx context.Context, satelliteID storj.NodeID, pieceID storj.PieceID) error
 	// DeleteFailed marks piece deletion from disk failed
 	DeleteFailed(ctx context.Context, satelliteID storj.NodeID, pieceID storj.PieceID, failedAt time.Time) error
+	// CachedSpaceUsed returns the in memory value for disk space used by all pieces
+	CachedSpaceUsed(ctx context.Context) (int64, error)
```

Comment:
Can we make this `SpaceUsed` and the other one `CalculateSpaceUsed` or similar? That way the call sites don't have to change.

Reply:
updated